toolchain/BUILD

```diff
@@ -0,0 +1,10 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# A convenience target for running the toolchain with the full prelude
+# available.
+alias(
+    name = "carbon",
```

Comment:
Did you consider naming this `toolchain` instead, to support simply `bazel run //toolchain`? I don't feel at all strongly about this, but the shorter name seems slightly nicer. But perhaps using the intended final name of the binary here is a good thing?

Reply:
Done.

I was hesitant similar to how it was "//toolchain/driver:carbon" instead of either "//toolchain/driver" or "//toolchain/carbon", but don't care strongly.